debugger/symbols.mli

```diff
@@ -69,3 +69,13 @@ val event_near_pos : string -> int -> code_event
 
 (* Recompute the current event *)
 val update_current_event : unit -> unit
+
+(* [bppm_matching_dirs path] gives the expansion of
+   path according to the BUILD_PATH_PREFIX_MAP spec.
+   Made available externally for unit testing, but also
+   used by the debugger. *)
+val bppm_matching_dirs : string -> string list
```

Comment:
I'm not fond of exposing this, even for testing. (The fact that we need the awkward `bppm` prefix suggests that abstractions are leaking here.) Could you instead expose a `get_program_source_dirs` function, and possibly `add_program_source_dir` as well? This way you could write your test by adding a directory and observing the rewritten matching directories.

Reply:
I have removed it. The unit tests now directly access the functions in Location, or Load_path.